rust/main/chains/hyperlane-cosmos/src/providers/cosmos/provider.rs

```diff
@@ -22,7 +22,8 @@ use crate::libs::account::CosmosAccountId;
 use crate::providers::rpc::CosmosRpcClient;
 use crate::{ConnectionConf, CosmosAmount, HyperlaneCosmosError, Signer};
 
-static MICRO_TO_ATTO: Lazy<U256> = Lazy::new(|| U256::from(1_000_000_000_000u128));
+/// Exponent value for atto units (10^-18).
+static ATTO_EXPONENT: Lazy<u32> = Lazy::new(|| 18);
```

Comment:
nit: could make this a const `u32` now since there's no expression left to evaluate

Reply:
Fixed